keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -881,6 +887,73 @@ Other components:
 * check how the change of meaning of resource requests influence other
   Kubernetes components.
 
+### Instrumentation
+
+The kubelet will record the following metrics:
+
+#### `kubelet_container_resize_requests_total` 
+
+This metric tracks the total number of resize requests observed by the Kubelet, counted at the container level.
+A single pod update changing multiple containers will be considered separate resize requests.
+
+Labels: 
+- `resource_type` - what type of resource is being resized. Possible values: `cpu_limits`, `cpu_requests` `memory_limits`, or `memory_requests`. If more than one of these resource types is changing in the resize request, 
+we increment the counter multiple times, once for each. This means that a single container update changing multiple
+resource types will be considered multiple requests for this metric. 
+- `operation_type` -  whether the resize is an increase or a decrease. Possible values: `increase`, `decrease`, `add`, or `remove`.
+- `namespace` - the namespace of the pod.
+
+This metric is recorded as a counter.
+
+#### `kubelet_pod_resize_sli_duration_seconds`
+This metric tracks the latency between when the kubelet accepts a resize request and when it finshes actuating the request. More precisely, this metric tracks the total amount of time that the PodResizeInProgress condition is present on a pod.
+
+Labels: 
+- `namespace` - the namespace of the pod.
+
+This metric is recorded as a gauge.
+
+#### `kubelet_pod_pending_resize_total`
+
+This metric tracks the total count of pods that the kubelet marks as pending. This will make it
+easier for us to see which of the current limitations users are running into the most.
+
+Labels:
+- `reason` - why the resize is pending. Possible values: `infeasible` or `deferred`.
+- `message` - more details about why the resize is pending. Although a more detailed "message" will be provided in the `PodResizePending`
```

Comment:
nit: message looks weird in a metric label. Maybe `reason_detail`?

Alternatively, call this reason, and rename `reason` above to something like `type` (although this doesn't match the condition)

Reply:
thank you :) I was struggling with what to call this